experanto/interpolators.py

```diff
@@ -140,113 +149,154 @@ def normalize_data(self, data):
         return data
 
     def interpolate(self, times: np.ndarray) -> tuple[np.ndarray, np.ndarray]:
-
         valid = self.valid_times(times)
         valid_times = times[valid]
+
+        if len(valid_times) == 0:
+            return np.array([]), valid
```

Comment:
Do we want to add a warning here that no points are valid for the queried time?
like 

```suggestion
        if len(valid_times) == 0:
            warnings.warn('Sequence interpolation returns empty array, no valid times queried')
            return np.array([]), valid
```
+ `import warnings` in the beginning